source/main/config.js

```diff
@@ -173,7 +173,11 @@ export const BLOCK_REPLAY_PROGRESS_CHECK_INTERVAL = 1 * 1000; // 1 seconds | uni
 export const FALLBACK_TOKEN_METADATA_SERVER_URL =
   'https://metadata.cardano-testnet.iohkdev.io';
 
+export const MINIMUM_AMOUNT_OF_RAM_FOR_RTS_FLAGS = 16 * 1024 * 1024 * 1024; // 16gb RAM
```

Comment:
Is it worth mentioning that the value is in `bytes`?
Example:

`MINIMUM_AMOUNT_OF_RAM_BYTES_FOR_RTS_FLAGS`

Reply:
But does it mean we need to change these variables too, mentioning in the variable name that those values are milliseconds? 
![image](https://user-images.githubusercontent.com/4303099/150505580-7d8ab45b-607e-4c9d-851e-01f5440b307d.png)

Reply:
Yes, I'd do so but not in this PR. Otherwise we can agree on base units `ms` and `bytes` then it's ok.

It's up to the team's decision I think.